mscore/menus.cpp

```diff
@@ -1466,9 +1466,15 @@ PalettePanel* MuseScore::newLinesPalettePanel()
 
       TextLine* textLine = new TextLine(gscore);
       textLine->setLen(w);
-      textLine->setBeginText("VII");
+      textLine->setBeginText("Staff");
       textLine->setEndHookType(HookType::HOOK_90);
-      sp->append(textLine, QT_TRANSLATE_NOOP("Palette", "Text line"));
+      sp->append(textLine, QT_TRANSLATE_NOOP("Palette", "Staff Text line"));
+
+      TextLine* systemTextLine = new TextLine(gscore, true);
```

Comment:
I'm wondering where "systemTextLine" will be deleted from heap

Reply:
Nowhere, this is and stays in the palette. Similar as the other elements here.